drivers/modem/modem_cmd_handler.c

```diff
@@ -486,6 +486,20 @@ int modem_cmd_handler_update_cmds(struct modem_cmd_handler_data *data,
 	return 0;
 }
 
+int modem_cmd_send_data_nolock(struct modem_iface *iface,
+			       const uint8_t *buf, size_t len)
+{
+#if defined(CONFIG_MODEM_CONTEXT_VERBOSE_DEBUG)
+	if (len > 256) {
```

Comment:
Where does the 256 come from, is it documented somewhere in the logger? This sounds like a bug tbh.

Reply:
I have no idea TBH. I have not been able to find out what causes this

Reply:
Maybe related to this issue? https://github.com/zephyrproject-rtos/zephyr/issues/51820